sources/packages/web/src/views/aest/student/FileUploads.vue

```diff
@@ -168,23 +93,17 @@ export default defineComponent({
         snackBar.success(
           "The documents were submitted and a notification was sent to the student.",
         );
-        await loadStudentFileUploads();
+        studentFileUploads.value.loadStudentFileUploads();
```

Comment:
The same approach is also used for the modals along the code, that is why I am not going to say, don`t do it but,
IMO, it can cause confusion because someone looking at the setup method may not expect it.

Is there a reason to believe that this would be clear that the providing the method as a parameter to be invoked, as shown below?

![image](https://user-images.githubusercontent.com/61259237/233697891-81e0e409-e773-4297-b919-82556c9f3e28.png)

Reply:
Vue Docs: https://vuejs.org/guide/essentials/template-refs.html#ref-on-component

In the last section [Ref on Component], it discusses the approach I followed but then it also says that the approach "makes it easy to create tightly coupled implementation details between the parent and the child, so component refs should be only used when absolutely needed - in most cases, you should try to implement parent / child interactions using the standard props and emit interfaces first."

Reply:
IMO, executing an operation/method of child component in parent looks like unorthodox, as mentioned in the Vue documentation it is used for tightly coupled implementation but is that what we want? we could follow the pattern what we followed for other components where we block these ones with v-if inside the component.

Reply:
Yes, I am reverting back that change.